package.json

```diff
@@ -3,11 +3,12 @@
   "version": "0.1.0",
   "private": true,
   "scripts": {
-    "serve": "vue-cli-service serve",
+    "serve": "(npm run cityzen-server || (sleep 10 && echo -e \"\\e[41mrun npm install -g codeforlansing/cityzen-server to enable autolaunching the backend\\e[0m\")) & vue-cli-service serve",
```

Comment:
Installing `npm-run-all` and using `run-p` may help simplify running both commands in parallel. Also, what do you think about leaving the `serve` script as `vue-cli-service` and introduce a `start` script that runs both `serve` and `cityzen-server` in parallel?

Reply:
Having this be the default with `start` seems reasonable, and also won't break stuff the depends on specific vue tooling

Reply:
And `npm-run-all` looks like it does exactly what I want! thanks